src/providers/DevicesProvider/docs/AudioInputProvider.stories.mdx

```diff
@@ -22,6 +22,12 @@ The `AudioInputProvider` provides a list of the user's available `audio-input` d
 
 ```
 
+NOTE: In Chrome, there is a "default" device. If a user chooses the "default" device as the current audio input device, and plugs in a new device during the 
+meeting, the new device will become the current audio input device also the new "default" device. But `selectedDevice` will not be updated since `selectedDevice` 
+stands for the deviceId, and the value of deviceId remains the same, still being `default`. If you want to get the update when switching to a new audio device, 
```

Comment:
"and the value of deviceId remains the same, still being `default`"  I think we should remove ",still being `default`" as it is a little confusing as if new device is updated as "default" why "selectedDevice" still being default is not updated.

Reply:
Maybe I need to add more details about it so builders can understand why `selectedDevice` is still being default.